trade-finance-manager-ui/templates/utilisation-reports/_macros/premium-payments-table.njk

```diff
@@ -0,0 +1,70 @@
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% import "./fee-record-status.njk" as feeRecordStatus %}
+{% import "../../_macros/table-cell-checkbox.njk" as tableCellCheckbox %}
+
+{% macro render(params) %}
+  {% set feeRecords = params.feeRecords %}
+
+  <table class="govuk-table" data-module="moj-sortable-table" data-cy="premium-payments-table">
+    <thead class="govuk-table__head">
+      <tr class="govuk-table__row">
+        <th scope="col" class="govuk-table__header"></th>
+        <th scope="col" class="govuk-table__header" aria-sort="ascending">Facility ID</th>
+        <th scope="col" class="govuk-table__header" aria-sort="none">Exporter</th>
+        <th scope="col" class="govuk-table__header" aria-sort="none">Reported fees</th>
+        <th scope="col" class="govuk-table__header" aria-sort="none">Reported payments</th>
+        <th scope="col" class="govuk-table__header" aria-sort="none">Total reported payments</th>
+        <th scope="col" class="govuk-table__header" aria-sort="none">Payments received</th>
+        <th scope="col" class="govuk-table__header" aria-sort="none">Total payments received</th>
+      <th scope="col" class="govuk-table__header">Status</th>
+      </tr>
+    </thead>
+
+    <tbody class="govuk-table__body">
+    {% for feeRecord in feeRecords %}
+      {% set checkboxId = ['facilityId-', feeRecord.facilityId] | join %}
+      <tr
+        class="govuk-table__row"
+        data-cy="premium-payments-table-row-{{ loop.index - 1 }}-facilityId-{{ feeRecord.facilityId }}"
```

Comment:
I believe we can have the same facility appearing multiple times in a report and therefore multiple times in the table. We might want to surface the fee record id as well but we should leave that to the tickets covering add a payment functionality to decide since it is possibly only of importance to those tickets? And so can perhaps leave this too for now. We probably want to explicitly include duplicates facility ids in some of our test data somewhere, perhaps e2e, but again I think we can safely leave that for now if you agree?

Reply:
I realised this was a problem which is why I added the `loop.index - 1` to `data-cy`, but after reading this I actually think it makes more sense to use the `feeRecordId` as we are likely to want that when we query things in future, so I will update it to use that